app/views/components/InputDateTime.scala.html

```diff
@@ -52,21 +52,21 @@
                 name = "dateDay",
                 label = Some(messages("date.day.capitalized")),
                 value = form("dateDay").value,
-                id = "dateDay"
+                id = "value.day"
```

Comment:
Is the `.withDayMonthYearWithErrorAsHtml` helper method usable for the day/month/year input items?

Reply:
I wasn't too sure on that - this is the one I messaged you about on thursday. With it being part of a different component, I wasn't sure if it would end up breaking the time aspect as that wasn't supported. I'll have a play around with it and see how I get on